nft-provider-aggregator/transactions/exampleNFT/revoke_withdraw_cap.cdc

```diff
@@ -0,0 +1,26 @@
+import NonFungibleToken from "NonFungibleToken"
+import ExampleNFT from "ExampleNFT"
+import NFTProviderAggregator from "NFTProviderAggregator"
+
+// Delete all withdraw capabilities targeting the default ExampleNFT collection storage path
+transaction() {
+    var countDeleted: Int
+    prepare(
+        signer: auth(LoadValue, Capabilities) &Account,
+    ) {
+        self.countDeleted  = 0
+        for controller in signer.capabilities.storage.getControllers(forPath: ExampleNFT.CollectionStoragePath) {
```

Comment:
To revoke the withdraw cap ("unlink" in pre-Cadence 1.0), we iterate over all capability controllers for the collection storage path, and delete those with the `nft-provider-aggregator` tag (set with the add nft withdraw cap txs), after validating the type as extra precaution.

Reply:
made the tag an argument, seems cleaner - thanks for suggesting it!